tests/TestCase/Core/ConfigureTest.php

```diff
@@ -69,6 +69,26 @@ public function tearDown()
     }
 
     /**
+     * testGet method
+     *
+     * @return void
+     */
+    public function testGet()
+    {
+        $expected = 'ok';
+        Configure::write('This.Key.Exists', $expected);
+        $result = Configure::get('This.Key.Exists');
+        $this->assertEquals($expected, $result);
+
+        try {
```

Comment:
I will separate the tests.
I don't really know how I should do it without try/catch.